core/src/main/java/com/linecorp/armeria/client/HttpClientPipelineConfigurator.java

```diff
@@ -227,6 +229,16 @@ private void configureAsHttps(Channel ch, SocketAddress remoteAddr) {
             sslHandler = sslCtx.newHandler(ch.alloc());
         }
 
+        p.addLast(new ChannelInboundHandlerAdapter() {
```

Comment:
How about not adding this handler when `fastOpen` is enabled?

Reply:
I think it would be good to add code for readability!! i will add `fastOpen` check code.